package.json

```diff
@@ -26,7 +26,7 @@
     "less": "~2.6.0",
     "less-loader": "~2.2.3",
     "ng-cache-loader": "0.0.16",
-    "phantomjs-prebuilt": "~2.1.14",
+    "phantomjs-prebuilt": "2.1.7",
```

Comment:
Let's use `2.1`.  This will give us the latest phantomjs 2.1.X with the latest phantomjs-prebuilt 2.1.Y installer.